src/Git_Updater/Base.php

```diff
@@ -520,7 +520,7 @@ public function upgrader_source_selection( $source, $remote_source, $upgrader, $
 		$new_source = $this->fix_misnamed_directory( $new_source, $remote_source, $upgrader_object, $slug );
 
 		if ( $source !== $new_source ) {
-			if ( function_exists( 'move_dir' ) ) {
+			if ( function_exists( __NAMESPACE__ . '\move_dir' ) ) {
```

Comment:
This is meant to be in the global namespace. There  will be a function `move_dir()` in core when https://core.trac.wordpress.org/ticket/51857 is committed.